checkout-service/client/src/components/AddToListDropDown.jsx

```diff
@@ -113,7 +113,7 @@ const generateListItems = () => {
   let listItems = [];
   for (let i = 0; i < randomNumOfLists; i++) {
     let randomPrivateOrPublicIndex = Math.floor(Math.random() * Math.floor(2));
-    listItems.push(<ListItem><ListItemImgWrapper><ListItemImg src={'https://placeimg.com/50/50'}></ListItemImg></ListItemImgWrapper><ListItemTextWrapper><ListItemLabel>{chance.word()}</ListItemLabel><ListItemStatus>{listStatus[randomPrivateOrPublicIndex]}</ListItemStatus></ListItemTextWrapper></ListItem>);
+    listItems.push(<ListItem key={uuidv4()}><ListItemImgWrapper><ListItemImg src={'https://placeimg.com/50/50'}></ListItemImg></ListItemImgWrapper><ListItemTextWrapper><ListItemLabel>{chance.word()}</ListItemLabel><ListItemStatus>{listStatus[randomPrivateOrPublicIndex]}</ListItemStatus></ListItemTextWrapper></ListItem>);
```

Comment:
So, up to this point I've been using the iteration index for my key. I read up on it and it seems as though is not the best practice because you end up with multiple children having the same key, provided you render children using more than one loop. There's apparently a benefit to each child having a unique key in the back-end of React. So, this package was recommended in the article to easily give each child a unique key.